app/scripts/templates/visits/item.html

```diff
@@ -1,4 +1,12 @@
-<div class="visit">
+<div class="image" style="background-image: url({{extractedImageUrl}});"></div>
```

Comment:
Instead of `<img src=""/>`? I think the CSS approach will prevent me from right-click+saving images...
Or not sure if there is a method we could use for lazy-loading images, if we didn't want to fire off 25+ network requests immediately (or whatever).